Add description (#519)
harness/developer-hub: docs/chaos-engineering/chaos-faults/kubernetes/pod/pod-dns-spoof.md

<details>
<summary>View fault usage</summary>
<div>

> This fault determines the resilience of an application to blocked DNS host names. It determines how quickly an application can resolve the host names and recover from the failure. It simulates custom responses from a spoofed upstream service. 

Reviewer: ```suggestion
This fault determines the resilience of an application when host names are resolved incorrectly. It determines how quickly an application can resolve the host names and recover from the failure. It simulates custom responses from a spoofed upstream service.
```
